ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/application-traffic-recognition.md

```diff
@@ -99,6 +103,8 @@ interface Management1
 | ---- | -------- |
 | dest_prefix_set1 | 2.3.4.0/24 |
 | dest_prefix_set2 | 4.4.4.0/24 |
+| empty-ipv4-prefixes |  |
```

Comment:
My suggestions should fix these blank fields.

Reply:
[7570f90](https://github.com/aristanetworks/ansible-avd/pull/3504/commits/7570f906d9b50a4a1676a765ea88fd3a971b4966)